Examples/BufferedExchange/src/BufferedExchangeProgram.cpp

```diff
@@ -0,0 +1,35 @@
+﻿#include <BufferedExchangeProgram.hpp>
+#include "Arp/System/Commons/Logging.h"
+#include "Arp/System/Core/ByteConverter.hpp"
+
+namespace BufferedExchange
+{
+
+    void BufferedExchangeProgram::Execute()
+    {
+        //implement program
+        if (count % 1000 == 0)
+        {
+            log.Info("---------------- Execute: {0} ", count);
+        }
+        count++;
+
+        //Watch out blocking!
+      //  if (bufferedExchangeComponent.wT.SetData(count))

```

Comment:
Enable setting data for WorkerThread
```suggestion
       if (bufferedExchangeComponent.wT.SetData(count))

```